flang/lib/Evaluate/intrinsics.cpp

```diff
@@ -2610,7 +2620,7 @@ bool IntrinsicProcTable::Implementation::IsDualIntrinsic(
     const std::string &name) const {
   // Collection for some intrinsics with function and subroutine form,
   // in order to pass the semantic check.
-  static const std::string dualIntrinsic[]{{"etime"}, {"getcwd"}};
+  static const std::string dualIntrinsic[]{{"etime"}, {"getcwd"}, {"rename"}};
```

Comment:
As long as you're changing this line, consider using `{"etime"s, "getcwd"s, "rename"s};` instead.

Reply:
Done.